### ISSUE TITLE
Ads plugin bug

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#7262]
There is a bug in the AD_Server plugin
When the advertisement video is shown at the beginning of the video, both videos play together at the same time
and you hear them both together.

In practice, you see the advertisement and hear the sound of the video on which the advertisement appears
in the background.

I currently can only check on the main site which has ads so I cannot reproduce on other sites.

**Comment by DanielnetoDotCom** [comment]
I notice that in some sites. the problem is that I cannot reproduce that in all sites.

try to disable the autoplay and check if it persists

**Comment by walt93** [comment]
FWIW I've seen the same thing. Ad server plays concurrent with video.  That's #1

#2 The ads came pre-configured to fire at some weird times like 3,10,60 iirc (seconds), so the user clicking and seeking would always fire an ad. Or two. We got a lot of user complaints about that one.

After serving 4,000-5,000 ads, the ad server falls down and stops serving ads, plays a video loop giving a completely incorrect error message (complaining about video file format - wot?) and can't be healed - you have to create a second campaign and run it until it fails, rinse, repeat.

Basically you can't use the ad server for anything real unless you're always monitoring it. Sorry to say... We're looking at Revive adserver with a $600 plugin to serve VAST ads, going to test it on our Roku channel next week………

**Comment by guymass** [comment]
OK after turning off the autoplay this issue is resolved. We will check in the future updates if this has stoped also while autoplay is on.
